backend/src/MyRoom.ts

```diff
@@ -132,8 +121,17 @@ export class MyRoom extends Room {
             let card = deck.takeTopCard();
 
             if (card !== undefined) {
-                this.state.table.addNewCard(
+                //Let the player drag this card
+                let player = this.state.getPlayer(client.sessionId);
+                if (player == null) {
```

Comment:
I think it's best style to always use ```===``` in javascript, especially when comparing things of different types

```suggestion
                if (player === null) {
```